CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Fixed
+- Fix the alignment of the collector and validate function (#255)
+
+### Changed
+- Bump the MSRV to 1.66.0 (#255)
```

Comment:
Bump the MSRV to allow using `AVec`. The dependency `aligned-vec -> equator-macro` requires the feature from new version.